server/aap/io/feeding_services/cision.py

```diff
@@ -0,0 +1,338 @@
+        "ENI": "06000000",  # Environmental Issues
+        "ENP": "06000000",  # Environmental Policy
+        "ESG": "06000000",  # Environmental, Social, and Governance
+        # "EGV": "",  #European Government
+        # "FDA": "",  #FDA Approval
+        # "FEA": "",  #Features
+        # "LEG": "",  #Federal and State Legislation
+        # "EXE": "",  #Federal Executive Branch, Agency News
+        # "FNC": "",  #Financing Agreements
+        "FOR": "11002002",  # Foreign policy/International affairs
+        # "FVT": "",  #Future Events
+        "DIS": "14004000",  # Handicapped/Disabled
+        # "HSP": "",  #Hispanic-Oriented News
+        # "IMA": "",  #International Medical Approval
+        # "INO": "",  #Investments Opinions
+        "JVN": "04016023",  # Joint Ventures
+        "LBR": "09010000",  # Labor/Union news
+        # "LAW": "",  #Legal Issues
+        "LGB": "14010001",  # Lesbian/Gay/Bisexual
+        "LIC": "04016026",  # Licensing/Marketing Agreements
+        "MRR": "04006008",  # Market Research Reports
+        # "MAT": "",  #Mature Audience
+        # "MAV": "",  #Media Advisory/Invitation
+        # "MIN": "",  #Mens Interest
+        # "ADM": "",  #MultiVu Audio
+        # "PHM": "",  #MultiVu Photo
+        # "VDM": "",  #MultiVu Video
+        # "NTA": "",  #Native American
+        "NAT": "03015001",  # Natural Disasters
+        "PDT": "04016030",  # New Products/Services
+        # "NPT": "",  #Not for Profit
+        # "OBI": "",  #Obituaries
+        # "OFR": "",  #Offerings
+        # "DSC": "",  #Oil/Gas Discoveries
+        "PLW": "04016031",  # Patent Law
+        # "PER": "",  #Personnel Announcements
+        "CPN": "11003002",  # Political Campaigns
+        # "PVP": "",  #Private Placement
+        "RCL": "04016053",  # Product Recalls
+        # "PSF": "",  #Public Safety
+        "RLE": "04004003",  # Real Estate Transactions
+        "REL": "12000000",  # Religion
+        "RCN": "04016039",  # Restructuring/Recapitalizations
+        # "SLS": "",  #Sales Reports
+        "SCZ": "14024005",  # Senior Citizens
+        # "SHA": "",  #Shareholder Activism
+        # "SHM": "",  #Shareholder Meeting
+        # "SRP": "",  #Shareholders' Rights Plans
+        # "SBS": "",  #Small Business Services
+        # "SRI": "",  #Socially Responsible Investing
+        "STS": "04016051",  # Stock Split
+        "SVY": "11003008",  # Surveys, Polls & Research
+        "TRD": "04008033",  # Trade Policy
+        # "TDS": "",  #Tradeshow News
+        # "STP": "",  #U.S. State Policy News
+        # "VEN": "",  #Venture Capital
+        "VET": "11001001",  # Veterans
+        # "VNR": "",  #Video News Releases
+        # "WOM": "",  #Women-related News
+    }
+
+    def _valid_token(self, provider):
+        if not provider.get("config", {}).get("token"):
+            return False
+
+        if utcnow() < arrow.get(provider.get("config", {}).get("token_expiry"), "YYYYMMDDTHHmmss").datetime:
```

Comment:
It might be good to wrap this in a `try..except` block and return `False`, which could happen if `token_expiry` is an invalid value for `arrow.get`.

Reply:
Done